test/subscriber.ts

```diff
@@ -52,7 +52,7 @@ const testRPC = new TestPolkadotRPC();
 
 let subject: Subscriber;
 
-function delay(ms: number) {
+function delay(ms: number): Promise<unknown> {
```

Comment:
Given that the promise is not resolving to any value, why not `Promise<void>` instead?

Reply:
it was auto inferred by my IDE ( visual studio code ). Let me quickly check